### DISCUSSION TITLE
seeing significant amount of expired keys after upgrade to 1.1.5 from 0.7.1

### DISCUSSION THREAD

**Discussion by jeremy-patreon** [sqlalchemy/dogpile.cache#216] [Usage Questions]
We canaried this upgrade and seeing significant amount of cache miss due to the expired key. All the keys grabbed by dogpile.cache are considered as expired keys. Could you shed some lights why this could happen?

**Comment by zzzeek** [comment]
the cache format changed in version 1.1 with the improvement described at https://dogpilecache.sqlalchemy.org/en/latest/changelog.html#change-ed02a6f3636a245baf548ba033853afa.  this bumped the version id of the cache that is stored with all values so running dogpile 1.1 or greater on a cache that was populated by a previous dogpile will see all existing keys as expired.
